src/security/KeyData.cc

```diff
@@ -89,6 +89,7 @@ void
 Security::KeyData::loadX509ChainFromFile()
 {
 #if USE_OPENSSL
+    // XXX: loadX509CertFromFile() has already opened and read this file.
```

Comment:
FTR: previous iteration of this comment said:
```
    // XXX: This BIO loads the public cert as first chain cert,
    //      so the code appending chains sends it twice in handshakes.
```
before @rousskov requested its removal as not being true.

Ironic that the bug claimed by this PR is:  "Configured chain R was sent as R-R instead of just R!"